script.js

```diff
@@ -19,48 +19,212 @@ for (i = 0; i < navLink.length; i++){
+const lastName= document.querySelector('.last-name');
+const comments = document.querySelector('.message');
+const email= document.querySelector('.email-address');
+function takeValuesfromInput(){
+    const firstNameValue = firstName.value;
+    const lastNameValue = lastName.value;
+    const commentsValue = comments.value;
+    const emailValue = email.value;
+    return { firstNameValue,lastNameValue,commentsValue,emailValue };
+}
+// SHOW ERRO FUNCTION
+function showEror(message,displayStyle){
+    const errHolder = document.createElement('p');
+    errHolder.classList="err-msg";
+    errHolder.innerHTML=message;
+    errHolder.style.display=displayStyle;
+    form.appendChild(errHolder);
+    setTimeout(() => {
+        errHolder.style.display='none';
+    }, 3000);
+}
+function clearForm(){
+    firstName.value='';
+    lastName.value='';
+    comments.value='';
+    email.value='';
+}
+form.addEventListener('submit',(e)=>{
+    e.preventDefault();
+   console.log('sumbit');
+    // take values
+    const {firstNameValue,lastNameValue,commentsValue,emailValue} = takeValuesfromInput(); 
+    saveInLocalStorage('data',{firstNameValue,lastNameValue,commentsValue,emailValue});
+    const urEmail = emailValue.toLowerCase();
+    console.log(urEmail,emailValue);
+    if(emailValue!==urEmail){
+        showEror("Email should be in lower case","block");
+    }
+    else if( firstNameValue==""||lastName==""||commentsValue==""||emailValue=="" ){
+        showEror("Inputs shouldn't be empty!","block");
+        console.log('eror');
+    }
+    else{
+        console.log('err');
+    }
+    // clear input
+    clearForm();
+})
+// day 4 local stroage
+function saveInLocalStorage(keyname,infos){
+    localStorage.setItem(keyname,JSON.stringify(infos));
+}
+function deleteDataFromStorage(){
+    localStorage.removeItem(data);
+}
+function checkingDataExistence(){
+    const storedDatas=localStorage.getItem('data');
+    if(storedDatas){
+        const infos = JSON.parse(storedDatas);
+        firstName.value=infos.firstNameValue;
+        lastName.value=infos.lastNameValue;
+        email.value=infos.emailValue;
+        comments.value=infos.commentsValue;
+        console.log(infos.firstNameValue);
+    }
+    else{
+        const {...datas} = takeValuesfromInput(); 
+        saveInLocalStorage('data',{...datas});
+        console.log('no data');
+    }
+}
+// delete data in localstroage
+resetBtn.addEventListener('click',(e)=>{
+    e.preventDefault();
+    clearForm();
+    localStorage.removeItem('data');
+})
+// const contactCover = document.querySelector('.contactSection');
+const popupCover = document.querySelector('.show-popup');
+function createElement(){
+    const card = document.createElement('div');
+    card.classList='showCard';
+    popupCover.append(card);
+
+}
+// download resume
+const downloadFile = document.querySelector('.downloadResume');
+downloadFile.addEventListener('click',()=>{
+    window.open("./README.md");
+})
```

Comment:
By referencing the changes requested by the previous code reviewer, you need to ensure that a file is downloaded when a user clicks the `Get my resume` button but currently, it's targeting to the readme file. Kindly fix this.